src/components/network-area-diagram-viewer/nad-viewer-parameters.ts

```diff
@@ -102,6 +102,12 @@ export interface NadViewerParametersOptions {
 
     // Size in pixel of the margin that is added to hoverable objects to help the user stay over them.
     hoverPositionPrecision?: number | null;
+
+    // Whether enabling adaptive zoom
```

Comment:
Could you describe the feature more precisely?

Reply:
the feature  description should be bettter, now